src/frontend/ui/capture/index.ts

```diff
@@ -548,15 +570,21 @@ export class Capture extends KojiBridge {
    * Prompts the user to select an image, file, audio, or video by selecting from the available asset packs, by uploading a file, or by entering a URL. Use this method to allow the user to select from more than one type of media with a single control. For example, allow the user to select an image or a video. You can limit the types of media to allow and configure options for each allowed type.
    *
    * @param   options
-   * @param   verbose Indicates whether to return additional metadata about the capture event. If `false` or not specified, returns only the value of the media capture, which is either the URL to the media as a string or an object with the URL and additional metadata.
-   * @return          [description]
+   * @param   verbose Indicates whether to return additional metadata about the capture event. If `false` or not specified, returns only the value of the media capture.
+   * @return          Value of the media capture, which is either the URL to the media as a string or an object with the URL and additional metadata, or the [[VerboseCapture]] object, if `verbose` is `true`.
    *
    * @example
    * ```javascript
    * const media = await Koji.ui.capture.media();
    *
-   * // Limit to image or video, hide asset packs, return an object with extended metadata, transcode videos for HLS
-   * const media = await Koji.ui.capture.media({ acceptOnly: [image,video], hideExtensions: true, returnType: 'extended', videoOptions: { hls: true } }, true);
+   * // Limit to image or video, hide asset packs,
+   * // return an object with extended metadata, transcode videos for HLS
+   * const media = await Koji.ui.capture.media({
+   *    acceptOnly: [image, video],
+   *    hideExtensions: true,
+   *    returnType: 'extended',
```

Comment:
`returnType` isn't accepted as a property anymore, just need to pass `true` as the second (verbose) argument to `Koji.ui.capture.media`.

Reply:
Done!